caikit/interfaces/common/data_model/stream_sources.py

```diff
@@ -30,6 +30,13 @@ class FileReference(DataObjectBase):
     filename: Annotated[str, FieldNumber(1)]
 
 
+@dataobject(PACKAGE_COMMON)
+class PathReference(DataObjectBase):
```

Comment:
This is a bit confusing because it's not really for a `DataStreamSource`

Reply:
The pattern that would follow how we do things currently would be to put this in `core.data_model.<somefile>.py`